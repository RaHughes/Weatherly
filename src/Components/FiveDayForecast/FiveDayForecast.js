import React from 'react';
import Card from '../Card/Card'

const FiveDayForecast = ({list}) => {
  console.log(list)
  const fiveDay = [list[1], list[9], list[17], list[25], list[33]]
  
  return (
    <Card data={fiveDay}/>
  )
}

export default FiveDayForecast
