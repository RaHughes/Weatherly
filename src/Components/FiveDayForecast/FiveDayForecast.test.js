import React from 'react';
import { shallow } from 'enzyme';
import FiveDayForecast from './FiveDayForecast';

describe('FiveDayForecast', () => {
  it('Should match the Snapshot', () => {
    const mockData = [{}, { one: 1 }, {}, {}, {}, {}, {}, {}, { two: 2 }, {},{}, {}, {}, {}, {}, {}, { three: 3 }, {}, {}, {}, {}, {}, {}, {}, { four: 4 }, {}, {}, {}, {}, {}, {}, {}, { five: 5} ]
    const wrapper = shallow(<FiveDayForecast list={mockData}/>)
    expect(wrapper).toMatchSnapshot()
  })
})