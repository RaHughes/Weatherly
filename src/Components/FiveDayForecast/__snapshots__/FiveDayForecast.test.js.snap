// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FiveDayForecast Should match the Snapshot 1`] = `ShallowWrapper {}`;
